cross-chain/arbitrum/README.adoc

```diff
@@ -0,0 +1,42 @@
+:toc: macro
+
+= Threshold cross-chain - Arbitrum
+
+This package brings Bitcoin to Ethereum L2 Arbitrum. For more details please
+see link:https://github.com/keep-network/tbtc-v2/blob/main/docs/rfc/rfc-8.adoc[Cross-chain Tokenized Threshold BTC]
+
+== How it works?
+
+```
++----------------------------+         +---------------------------------------------------------------------------+
+|          Ethereum          |         |                                Arbitrum                                   |
+|                            |         |                                                                           |
+|  +----------------------+  |         |  +----------------------+  +-------------------------+  +--------------+  |
+|  | Wormhole TokenBridge |--|---------|--| Wormhole TokenBridge |--| ArbitrumWormholeGateway |--| ArbitrumTBTC |  |
+|  +----------------------+  |         |  +----------------------+  +-------------------------+  +--------------+  |
+|                            |         |                                                                           |
++----------------------------+         +---------------------------------------------------------------------------+
+```
+
+- `ArbitrumTBTC` canonical tBTC token on Arbitrum with a minting authority
+delegated to `ArbitrumWormholeGateway`.
+- `ArbitrumWormholeGateway` is a smart contract wrapping and unwrapping 
+Wormhole-specific tBTC representation into the canonical `ArbitrumTBTC` token.
+
+=== Deploy contracts
+
```

Comment:
Can we mention that this module doesn't deploy any contracts to L1 as it uses the Warmhole's instances stored in the `external/goerli` and `external/mainnet` directories?